pkgs/applications/graphics/hdrmerge/default.nix

```
@@ -0,0 +1,77 @@
+{ lib
+, mkDerivation
+, fetchpatch
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, qtbase
+, wrapQtAppsHook
+, libraw
+, exiv2
+, zlib
+, alglib
+, pkg-config
+, makeDesktopItem
+}:
+
+mkDerivation rec {
+  pname = "hdrmerge";
+  version = "unstable-2020-11-12";
+  src = fetchFromGitHub {
+    owner = "jcelaya";
+    repo = "hdrmerge";
+    rev = "f5a2538cffe3e27bd9bea5d6a199fa211d05e6da";
+    sha256 = "1bzf9wawbdvdbv57hnrmh0gpjfi5hamgf2nwh2yzd4sh1ssfa8jz";
+  };
+
+  # dev dependencies
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    wrapQtAppsHook
+  ];
+
+  # runtime dependencies
```

```suggestion
  nativeBuildInputs = [
    cmake
    pkg-config
    wrapQtAppsHook
  ];

```
Those comments are not accurate. nativeBuildInputs are things required in the arch that the build machine is on, buildInputs contains things in the arch that the final binary is in.